app/views/moves/print/header.html.slim

```diff
@@ -0,0 +1,35 @@
+doctype html
+
+javascript:
+  function number_pages() {
+    var vars={};
+    var x=document.location.search.substring(1).split('&');
```

Comment:
We should probably gives these `var`s better names.

<img src="http://www.pmslweb.com/the-blog/wp-content/uploads/2016/05/2-a-man-needs-a-name-funny-GoT-meme.jpg" />